projects/plugins/jetpack/tests/php/modules/masterbar/test-class-admin-menu.php

```diff
@@ -192,12 +194,11 @@ public function test_add_posts_menu() {
 	 * @covers ::add_media_menu
 	 */
 	public function test_add_media_menu() {
-		global $menu, $submenu;
+		global $menu;
 
 		static::$admin_menu->add_media_menu();
 
 		$this->assertSame( 'https://wordpress.com/media/' . static::$domain, $menu[10][2] );
-		$this->assertEmpty( $submenu['upload.php'] );
```

Comment:
Instead of removing this assert, can we change it so we check there are no visible submenus?

Reply:
Good point, thanks! :)